pkg/apis/pipeline/v1/pipeline_types.go

```diff
@@ -114,7 +115,6 @@ type PipelineSpec struct {
 	Tasks []PipelineTask `json:"tasks,omitempty"`
 	// Params declares a list of input parameters that must be supplied when
 	// this Pipeline is run.
-	// +listType=atomic
```

Comment:
`Params` is of type `ParamsSpec`, not a list of something, so `+listType=atomic` does not apply, right? Is this the reason for removing?